packages/react-native-tracer-provider/src/util.ts

```diff
@@ -0,0 +1,55 @@
+import { Attributes, TimeInput } from '@opentelemetry/api';
+import { Link } from '@opentelemetry/api/build/src/trace/link';
+
+/**
+ * Several different primitive types are valid as TimeInput, normalize to a number representing epoch milliseconds
+ * to simplify communication with the native modules
+ *
+ * See https://reactnative.dev/docs/native-modules-android#argument-types and https://reactnative.dev/docs/native-modules-ios#argument-types
+ */
+export const normalizeTime = (time?: TimeInput): number => {
+  if (time === undefined) {
+    return 0;
+  }
+
+  if (typeof time === 'number') {
+    return time;
+  }
+
+  if (time instanceof Date) {
```

Comment:
Why are type checking semantics on 19 and 15 different?

Reply:
`typeof` only works for primitive types since Date is a class typeof would just return "object" without giving more info so need `instanceof` to see that it is actually a Date